src/lib.rs

```diff
@@ -1,14 +1,34 @@
-use red4ext_rs::prelude::*;
+use red4ext_rs::{prelude::*};
 
 define_plugin! {
     name: "example",
     author: "jekky",
     version: 1:0:0,
     on_register: {
         register_function!("SumInts", sum_ints);
+        register_function!("PluginName", plugin_name);
+        register_function!("Convert", convert);
+        register_function!("Append", append);
     }
 }
 
 fn sum_ints(ints: Vec<i32>) -> i32 {
     ints.iter().sum()
 }
+
+fn plugin_name() -> String {
+    String::from("RED4EXT.RS.EXAMPLE")
+}
+
+fn convert(name: String) -> TweakDBID {
+    TweakDBID::new(&name)
+}
+
+
+/// test in CET like:
+/// ```swift
+/// LogChannel(CName.new("DEBUG"), TDBID.Create("A.Test") == Append(TDBID.Create("A."), "Test"));
+/// ```
+fn append(base: TweakDBID, suffix: String) -> TweakDBID {
```

Comment:
```suggestion
fn append_to_tweakdb_id(base: TweakDBID, suffix: String) -> TweakDBID {
```

Reply:
done in [74b89e2](https://github.com/jac3km4/red4ext-rs-example/pull/1/commits/74b89e26d0a5ad5b63ef6ab561039d07adfc84a6)